### DISCUSSION TITLE
Hi, I need little bit help..for simple stream example

### DISCUSSION THREAD

**Discussion by Kiwi915** [pschatzmann/arduino-audio-tools#591] [Q&A] [answered]
Hi there,
Can anybody advise me how to make a first sample for a stream with using a external DAC like 5102a.
There are so many examples ,but i think I do not found the right one in all folders.

I was looking for a simple example for a first simple Webstream by http:// Adress and output over the I2S to a pcm5102a.

Many thanks for helping...
Kind regards

**Comment by pschatzmann** [comment, marked as answer]
I am not sure what you mean: You can find connection information here https://github.com/pschatzmann/arduino-audio-tools/wiki/External-DAC. After you set up the hardware you use the I2SStream as explained in the documentation

An introduction in what audio samples are and how to write them is here: https://github.com/pschatzmann/arduino-audio-tools/wiki/Introduction

Maybe you just read the Wiki!
